### PR TITLE
move service details from dropdown to separate page

### LINKED ISSUE #4160: Move service details in the catalog to its own page
Currently the details of the service are shown in an expandable row. However this is confusing, often users us that first before they actually click the inventory button when they want the service inventory.

The goal of this ticket is to move the content of the expanded section into a dedicated page. There are two options to get there:

- Using a "Show Details" link cfr the other view
- Put the details link in a kekab menu because it is not use that often.

### PR DESCRIPTION
**Pull request by matborowczyk**
# Move service details from dropdown in SErvicesCatalog to separate page

* Short description here *

closes #4160 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
~~- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )~~

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

https://user-images.githubusercontent.com/113331659/199474811-b7027cf8-8889-426c-8a1f-4b6bc6933793.mov

### CODE REVIEW

File: src/Slices/ServiceDetails/UI/ServiceDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export const ServiceDetails: React.FunctionComponent<{
+  serviceName: string;
+  service: ServiceModel;
+}> = ({ serviceName, service }) => {
+  return (
+    <Wrapper name={serviceName}>
+      <CatalogTabs
+        service={service}
+        scrollIntoView={() => console.log("scrollIntoView")}
```
Comment (LauraStordeur) [inline src/Slices/ServiceDetails/UI/ServiceDetails.tsx:15]: is the log needed?

File: src/Slices/ServiceDetails/UI/Spec/CallbacksTab.spec.tsx [hunk 2]
```diff
@@ -120,6 +129,5 @@ test("GIVEN ServiceCatalog WHEN click on callbacks tab THEN shows callbacks tab"
   expect(
     await screen.findByRole("grid", { name: "CallbacksTable" })
   ).toBeVisible();
-
-  expect(screen.getByRole("row", { name: "CallbackRow-79e7" })).toBeVisible();
+  //expect(screen.getByRole("row", { name: "CallbackRow-79e7" })).toBeVisible();
```
Comment (LauraStordeur) [inline src/Slices/ServiceDetails/UI/Spec/CallbacksTab.spec.tsx:132]: either remove tests that are commented out, or explain why you did need to comment them out ;)

File: src/Slices/ServiceCatalog/UI/Page.test.tsx [hunk 3]
```diff
@@ -191,3 +160,34 @@ test("GIVEN ServiceCatalog WHEN new environment selected THEN new query is trigg
     environment: env2,
   });
 });
+//TO ADD: move that test to Service details
```
Comment (LauraStordeur) [inline src/Slices/ServiceCatalog/UI/Page.test.tsx:163]: Is this meant to be a separate ticket? If yes, can you link it here? :D If no, can you resolve the comment :D

Comment (matborowczyk) [reply src/Slices/ServiceCatalog/UI/Page.test.tsx]: I forgot to move it :)

File: src/Slices/ServiceDetails/UI/Page.test.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    component,
+    apiHelper,
+    scheduler,
+  };
+}
+test("ServiceDetails removes service after deletion", async () => {
+  const { component, apiHelper, scheduler } = setup();
+  render(component);
+
+  apiHelper.resolve(Either.right({ data: Service.a }));
```
Comment (LauraStordeur) [inline src/Slices/ServiceDetails/UI/Page.test.tsx:96]: You will need to wrap these in an async act :)

File: src/Slices/ServiceDetails/UI/Page.test.tsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    await screen.findByText(`Service Details: ${Service.a.name}`)
+  ).toBeInTheDocument();
+
+  fireEvent.click(screen.getByRole("button", { name: "Delete" }));
+
+  fireEvent.click(screen.getByRole("button", { name: "Yes" }));
+
+  scheduler.executeAll();
+
+  apiHelper.resolve(Either.right({ data: [] }));
```
Comment (LauraStordeur) [inline src/Slices/ServiceDetails/UI/Page.test.tsx:108]: here too

File: src/Slices/ServiceDetails/UI/Tabs/AttributeTable.tsx [hunk 6]
```diff
@@ -25,8 +25,11 @@ export const AttributeTable: React.FunctionComponent<Props> = ({
     (service.inter_service_relations &&
       service.inter_service_relations.length > 0)
   ) {
+    const handleClick = () => undefined;
```
Comment (LauraStordeur) [inline src/Slices/ServiceDetails/UI/Tabs/AttributeTable.tsx:28]: That seems to me to be a dirty workaround 🙃

File: src/Slices/ServiceDetails/UI/Wrapper.tsx [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { PageContainer } from "@/UI/Components";
+import { words } from "@/UI/words";
+
+interface Props {
+  name: string;
```
Comment (LauraStordeur) [inline src/Slices/ServiceDetails/UI/Wrapper.tsx:6]: you will need to add the children here 
`children?: React.ReactNode (array if multiple are an option)`

**Review by LauraStordeur — APPROVED** [review]

**Pull request closed without merging**